openactive-dataset.gemspec

```diff
@@ -0,0 +1,34 @@
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "openactive/dataset/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "openactive-dataset"
+  spec.version       = OpenActive::Dataset::VERSION
+  spec.authors       = ["Joe Carter"]
+  spec.email         = ["joe@simpleweb.co.uk"]
+
+  spec.summary       = 'Write a short summary, because RubyGems requires one.'
```

Comment:
Update this summary - may want to ask Nick or Tim or copy from PHP/.NET

Reply:
@nickevansuk Small one for you. Can't see any existing package description for the .NET/PHP implementations.